lobsterpy/cohp/test/test_describe.py

```diff
@@ -400,78 +426,129 @@ def test_text(self):
             ],
         )
         self.assertEqual(
             self.describe_K3Sb_all.text,
             [
                 "The compound K3Sb has 3 symmetry-independent atoms(s) with relevant bonds: K1, K2, Sb4.",
-                "K1 has a 14-fold coordination environment. It has 8 K-K (mean ICOHP: -0.37 eV, 17.544 percent antibonding interaction below EFermi), and 6 K-Sb (mean ICOHP: -0.14 eV, 2.299 percent antibonding interaction below EFermi) bonds.",
-                "K2 has a 14-fold coordination environment. It has 10 K-K (mean ICOHP: -0.22 eV, 17.073 percent antibonding interaction below EFermi), and 4 K-Sb (mean ICOHP: -0.36 eV, 4.969 percent antibonding interaction below EFermi) bonds.",
-                "Sb4 has a 14-fold coordination environment. It has 14 Sb-K (mean ICOHP: -0.27 eV, 3.731 percent antibonding interaction below EFermi) bonds.",
+                "K1 has a 14-fold coordination environment. It has 8 K-K (mean ICOHP: -0.37 eV, 17.544 percent "
+                "antibonding interaction below EFermi), and 6 K-Sb (mean ICOHP: -0.14 eV, 2.299 percent "
+                "antibonding interaction below EFermi) bonds.",
+                "K2 has a 14-fold coordination environment. It has 10 K-K (mean ICOHP: -0.22 eV, 17.073 "
+                "percent antibonding interaction below EFermi), and 4 K-Sb (mean ICOHP: -0.36 eV, 4.969 "
+                "percent antibonding interaction below EFermi) bonds.",
+                "Sb4 has a 14-fold coordination environment. It has 14 Sb-K (mean ICOHP: -0.27 eV, 3.731 "
+                "percent antibonding interaction below EFermi) bonds.",
             ],
         )
         self.assertEqual(
             self.describe_CsH_all.text,
             [
                 "The compound CsH has 2 symmetry-independent atoms(s) with relevant bonds: Cs1, H2.",
-                "Cs1 has a 18-fold coordination environment. It has 18 Cs-Cs (mean ICOHP: -0.49 eV, 18.741 percent antibonding interaction below EFermi) bonds.",
+                "Cs1 has a 18-fold coordination environment. It has 18 Cs-Cs (mean ICOHP: -0.49 eV, 18.741 "
+                "percent antibonding interaction below EFermi) bonds.",
                 "H2 has a 0 coordination environment. It has 0 bonds.",
             ],
         )
+        self.assertEqual(
+            self.describe_BaTiO3_orb.text,
+            [
+                "The compound BaTiO3 has 1 symmetry-independent cation(s) with relevant cation-anion interactions: Ti2.",
+                "Ti2 has an octahedral (CN=6) coordination environment. It has 6 Ti-O (mean ICOHP: -3.54 eV, "
+                "1.092 percent antibonding interaction below EFermi) bonds.",
+                "The orbitals 2s-4s, 2s-3dxy, 2py-3dxy, 2px-3dx2, 2px-3dxy, 2py-3dyz, 2s-3dz2, 2pz-3dz2, "
+                "2px-3dxz, 2pz-4s contribute 21.79, 11.4, 13.1, 10.76, 13.6, 14.56, 15.26, 25.5, 13.75, "
+                "11.5 percent in the Ti-O bond, respectively.",
```

Comment:
It's somehow clear that this isn't sorted by atom type. 2s clearly belongs to O and not Ti. I am also not sure this is the most useful output. You could add the most important orbital-interaction to the brackets before. It's going to be an extremely long list otherwise.

Reply:
Fixed this now 👍